unitTests/ui/ResultFoldingTest.ts

```diff
@@ -358,5 +380,33 @@ export function ResultFoldingTest() {
         done();
       });
     });
+
+    it('should call Quickview open when ResultLink has openQuickview option', done => {
+      const result = FakeResults.createFakeResultWithChildResult('foo', 1);
+      result.childResults[0].flags = 'HasHtmlVersion';
+
+      test = Mock.optionsResultComponentSetup<ResultFolding, IResultFoldingOptions>(
+        ResultFolding,
+        <IResultFoldingOptions>{
+          resultTemplate: UnderscoreTemplate.fromString(
+            '<div class="CoveoQuickview"></div><a class="CoveoResultLink" data-open-quickview="true"/>',
+            {}
+          )
+        },
+        result
+      );
+
+      Defer.defer(() => {
+        const quickview = $$(test.cmp.element).find('.CoveoQuickview');
+        const resultLink = $$(test.cmp.element).find('.CoveoResultLink');
+        const spyOnQuickviewOpen = jasmine.createSpy('openQuickview');
+        (<Quickview>get(quickview)).open = spyOnQuickviewOpen;
+
+        resultLink.click();
+
+        expect(spyOnQuickviewOpen).toHaveBeenCalled();
+        done();
+      });
```

Comment:
I think it's important if we could also add a test to check that `event.stopPropagation()` is called when we click.
You may to trigger the event differently. Instead of `resultLink.click()`, you have to do something like:

```
const click = new Event('click');
<targetElement>.dispatchEvent(click);

expect(click.stopPropagation).toHaveBeenCalledTimes(1);
```